docs/home/contribute/includes.md

```diff
@@ -1,12 +1,66 @@
 ---
 approvers:
 - chenopis
-title: Tabs Example
+title: Standard Include Statements
 ---
 
+{% capture overview %}
+This page explains the special include statements that can be used in
+Kubernetes documentation markdown.
```

Comment:
could you mention that "includes" is a built-in Jekyll function and link out to https://jekyllrb.com/docs/templates/#includes (for the sake of potential new contributors)?

Reply:
:white_check_mark: